docs/source/gbd2017_models/risks/low_birthweight_short_gestation/index.rst

```diff
@@ -51,3 +52,162 @@ described below, and relative to the joint TMREL.
+|  382     | neonatal encephalopathy due to birth asphyxia and trauma|
++----------+---------------------------------------------------------+
+|  383     | neonatal sepsis and other neonatal infections           |
++----------+---------------------------------------------------------+
+|  384     | hemolytic disease and other neonatal jaundice           |
++----------+---------------------------------------------------------+
+|  385     | other neonatal disorders                                |
++----------+---------------------------------------------------------+
+|  686     | sudden infant death syndrome                            |
++----------+---------------------------------------------------------+
+
+
+Vivarium modelling strategy
++++++++++++++++++++++++++++
+
+First, we convert the GBD 500g-2weeks birthweight-ga bins/categories to a joint continuous distribution using Abie's notebook (add link?). We assume a uniform distribution within each bin/category. 
+
+.. note ::
+    That this is likely biasing towards overestimating extreme birthweights or gestational ages. For example, in the 0-500g category, most babies are probably pretty close to 500g, not equally probable to be 1 gram versus 499 grams.
+
+Because the relative risks from GBD are for all-cause mortality in the early and late neonatal period, we first define all-cause mortality rate (ACMR) as the sum of: 
+
+   - mortality from causes that are affected by lbwsg and modelled in the sim (green)
+   - mortality from causes that are affected by lbwsg but not modelled in the sim (blue)
+   - mortality from causes that are unaffected by lbwsg, modelled or not modelled (salmon)
+
+An example of these causes from the large-scale-food fortification concept model diagram is shown below: 
+
+.. image:: causes_equation.svg
+
+All-cause mortality is sum of all cause-specific mortalities: 
+
+   ACMR =  :math:`\sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{blue}}\text{CSMR} + \sum\limits_{\text{salmon}}\text{CSMR}`
+
+The mortality from unmodelled causes affected by lbwsg (blue) is thus: 
+
+   ACMR - :math:`\sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR}`
+
+We are interested in applying the PAF and relative risk to only the causes that GBD considers to be affected by lbwsg (green and blue):
+
+   |  i = low birth weight short gestation category
+   |  mr_i = mortality hazard in early and late neonatal period for category i
+   |  rr_i = relative risk for all cause mortality in category i
+   |  state = either 1 with condition or 0 without condition 
+   |  PAF* = this is the PAF of the most detailed cause affected by lbwsg
+
+Hence, the mortality hazard for an individual in category i is:  
+
+:math:`\sum\limits_{\text{salmon}}\text{CSMR} + (ACMR - \sum\limits_{\text{salmon}}\text{CSMR} - \sum\limits_{\text{green}}\text{CSMR} + \sum\limits_{\text{green}}\text{EMR_state})\cdot\text{(1-PAF*)}\cdot rr_i`
```

Comment:
I think it would be worth writing out a multi-line derivation of this equation, so the reader can easily follow all the steps and understand where each piece comes from. I'd be willing to work on that for a future PR -- I think the present version is fine for now.

Reply:
I attempted to write out how the equation is derived.

Reply:
Cool, thanks, that helps!

A question occurred to me while reading this:

Does this method require us to pull CSMRs for _every single cause_ in GBD _except_ for the "blue" ones (the un-modeled causes affected by LBWSG)? Because we're summing up CSMRs for all the "salmon" causes, which include _all_ of the unaffected causes that we are _not_ modeling? And if that's the case, why not just compute the sum of the blue CSMRs directly too, instead of computing it from ACMR, salmon, and green?

If we want to minimize the number of things we need to explicitly compute or pull data for, it seems like it would make more sense to solve for the salmon CSMRs in the original equation (instead of the blue ones), then write everything in terms of the green and blue CSMRs instead of the green and salmon ones.

Am I understanding this correctly?

Reply:
salmon includes all the unaffected causes that we are both modelling and not modelling. 

I guess we should use:

 cause_group_with_the_most_causes = ACMR - cause_groupS_with_the_least_causes 

to save on computation ? 

Not sure why we decided to derive blue causes from ACMR rather than the other ones...

Reply:
Yeah, I don't know.  I didn't totally follow the discussion between @alibow  and @collijk about the different equation options...

Reply:
I don't think we addressed this specific issue directly. I think that either way works and that whatever makes the most sense from a software perspective will work here.